features/cellular/framework/targets/UBLOX/N2XX/mbed_lib.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "UBLOX_N2XX",
+    "config": {
+        "tx": {
+            "help": "TX pin for serial connection. D1 assumed if Arduino Form Factor, needs to be set/overwritten otherwise.",
+            "value": null
+        },
+        "rx": {
+            "help": "RX pin for serial connection. D0 assumed if Arduino Form Factor, needs to be set/overwritten otherwise.",
+            "value": null
+        },
+        "rts": {
+            "help": "RTS pin for serial connection",
+            "value": null
+        },
+        "cts": {
+            "help": "CTS pin for serial connection",
+            "value": null
+        },
+        "baudrate" : {
+            "help": "Serial connection baud rate",
+            "value": 9600
+        },
```

Comment:
`baudrate` is suspiciously low?

Reply:
Default baud rate is correct here; on this device it allows the UART to run from 32 kHz crystal hence the device can run in lowest possible power mode (3 uA @ 3.3V) and still always be responsive to the host.